### ISSUE TITLE
Не вызывается метод index при использовании AJAX

### ISSUE THREAD

**Issue by dimabubenok** [64j/ModxLoader#1]
При использовании AJAX не вызывается метод index моего класса.
Причина в проверке на 54 строке
`if($method == 'index' && isset($_GET['route']))`
т.к. если делать AJAX-запрос на страницу ajax.php и передавать роут без указания метода через query-string т.е. так:
`ajax.php?route=test/controller/core`
то получается что у запроса есть GET часть route и метод будет index, значит проверка на 54 строке пройдет и плагин  сделает редирект...
Вышел из положения добавив в условие проверку метода запроса:
`if($method == 'index' && isset($_GET['route']) && $_SERVER['REQUEST_METHOD'] == "GET")`

**Comment by 64j** [comment]
так там всё через post отправляется)

**Comment by dimabubenok** [comment]
Да, отправляется через POST и тем не менее проверка на 54 строке проходит.
При этом `$_GET['route']` содержит query-string  `test/controller/core`,
а `$_SERVER['REQUEST_METHOD'] ` POST
